File: lab№5_1.py
import math
x = float(input("Введіть x: "))
if x > 1:
    y = math.log10(abs(x + 1)) + 2.9 * math.exp(0.1 * x)
elif x > -1.1:
    y = math.sqrt(abs(x)) + x ** (1 / 3) - math.sin(x)
else:
    y = 4 * x + math.exp(x) - 4 * math.sqrt(abs(x))
print("y =", y)

File: lab№5_1.cpp
#include <iostream>
#include <cmath>
using namespace std;
int main()
{
    double x, y;
   cout << "x = ";
   cin >> x;
    if (x > 1)
 {
    y = log10(abs(x + 1)) + 2.9 * exp(0.1 * x);
 }
  else if (x > -1.1)
     {
        y = sqrt(abs(x)) + cbrt(x) - sin(x);
    }
    else
    {
         y = 4 * x + exp(x) - 4 * sqrt(abs(x));
    }
    cout << "y = " << y;
     return 0;
}   
 
